### PR TITLE
fix: add status page test for rule 921120

### PR DESCRIPTION
**Pull request by RedXanadu**
PR adds a status page test for rule 921120 (a new test which only triggers this specific rule).

### CODE REVIEW

**Review by dune73 — APPROVED** [review]

**Comment by dune73** [comment]
Thank you @RedXanadu. This looks good.

Ready to merge once tests are passing.

**Comment by RedXanadu** [comment]
This was a copy-paste of the Status Page test template I've been using since the Dev Retreat. I don't understand why the tests are now failing. I'll dig into this when I have some time.

**Comment by fzipi** [comment]
The failing test was on the title of the PR itself. Remember we are using conventional commits. So there is a test for that. 🤷

File: tests/regression/tests/REQUEST-921-PROTOCOL-ATTACK/921120.yaml [hunk 1]
```diff
@@ -62,3 +62,22 @@ tests:
...
+            dest_addr: 127.0.0.1
+            headers:
+              Host: localhost
+              Proxy-Connection: keep-alive
+              Referer: http
+              User-Agent: OWASP ModSecurity Core Rule Set
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+            method: POST
+            port: 80
+            uri: "/"
```
Comment (fzipi) [inline tests/regression/tests/REQUEST-921-PROTOCOL-ATTACK/921120.yaml:79]:
```suggestion
            uri: "/post"
```

**Review by fzipi — CHANGES_REQUESTED** [review]
I'm suggesting to use `/post` when we are posting so logs are cleaner.

**Review by dune73 — APPROVED** [review]

**Comment by dune73** [comment]
I am submitted your suggestion @fzipi so we can merge this. Yet GH wants you to approve first.

Please approve and merge.

**Review by fzipi — APPROVED** [review]

**Pull request merged**